tests/controls/textentrytest.cpp

```diff
@@ -45,17 +45,25 @@ void TextEntryTestCase::TextChangeEvents()
     wxTextEntry * const entry = GetTestEntry();
 
     // notice that SetValue() generates an event even if the text didn't change
+#ifndef __WXQT__
     entry->SetValue("");
     CPPUNIT_ASSERT_EQUAL( 1, updated.GetCount() );
     updated.Clear();
+#else
+    WARN("Events are only sent when text changes in WxQt");
```

Comment:
I'm fine with not fixing it in this PR if it doesn't break it, i.e. if this already didn't work before, but I just wanted to say that this is a pretty bad problem as it will break plenty of existing code which often specifically ignores events from `SetValue()`, so not getting them can break the program logic in unexpected and difficult to understand ways.

It shouldn't be difficult to send the events from `SetValue()` in Qt too and it would be very much worth doing this IMO.

Reply:
Unfortunate wording in the warning. I agree it should be fixed. Will deal with that after the below.